apps/schema/static/schema/swagger.yaml

```
@@ -2191,6 +2468,115 @@ components:
             included:
               $ref: '#/components/schemas/shipment/included'
 
+    tracking:
+      collectionResource:
+        properties:
+          type:
+            example: FeatureCollection
+
+      singularResource:
+        properties:
+          type:
+            example: Feature
+
+      pointProperties:
+        type: object
+        properties:
+          properties:
+            type: object
+            allOf:
+            - $ref: '#/components/schemas/dataTypes/tracking/time'
+            - $ref: '#/components/schemas/dataTypes/tracking/uncertainty'
+            - $ref: '#/components/schemas/dataTypes/tracking/hasGPS'
+            - $ref: '#/components/schemas/dataTypes/tracking/source'
+
+      lineProperties:
+        type: object
+        properties:
+          properties:
+            type: object
+            allOf:
+            - $ref: '#/components/schemas/dataTypes/tracking/linestringTimestamps'
+
+      getPointResource:
+        type: array
+        items:
+          type: object
+          properties:
+            type:
+              type: string
+              example: FeatureCollection
+            features:
+              type: array
+              items:
+               $ref: '#/components/schemas/tracking/pointFeature'
+
+      pointResponse:
+        description: Response data with parameter as_point
+        allOf:
+        - properties:
+            data:
+              allOf:
```

Add a description to each of your Response's `data` objects like this:
```suggestion
              description: A GeoJSON FeatureCollection containing Point data.
              allOf:
```